document-warehouse/package.json

```diff
@@ -8,6 +8,7 @@
     },
     "dependencies": {
         "@google-cloud/contentwarehouse": "^0.5.1",
-        "assert": "^2.0.0"
+        "assert": "^2.0.0",
```

Comment:
Please move any dependencies that are only used for tests to a `devDependencies` section.